docs/wallet/address-tagging.md

```diff
@@ -0,0 +1,125 @@
+---
+id: address-tagging
+title: Address and UTXO tagging
+---
+
+```scala mdoc:invisible
+import org.bitcoins.core.wallet.utxo._
+import org.bitcoins.core.protocol.BitcoinAddress
+import org.bitcoins.core.protocol.transaction._
+import org.bitcoins.core.wallet.fee._
+import org.bitcoins.core.currency._
+import org.bitcoins.wallet.models.AccountDb
+import org.bitcoins.wallet._
+
+val ExampleAddressTag = CustomAddressTag("name", "tagType")
```

Comment:
we should probably give an example which extends `ExternalAddressTag` directly so that users have a good model no?

Reply:
This is invisible, just needed for the wallet api examples